examples/tools/scene_viewer.rs

```diff
@@ -285,14 +295,18 @@ fn camera_spawn_check(
 
 fn check_camera_controller(
     mut commands: Commands,
-    camera: Query<Entity, (With<Camera>, Without<CameraController>)>,
+    camera: Query<(Entity, &Transform), (With<Camera>, Without<CameraController>)>,
     mut found_camera: Local<bool>,
 ) {
     if *found_camera {
         return;
     }
-    if let Some(entity) = camera.iter().next() {
-        commands.entity(entity).insert(CameraController::default());
+    if let Some((entity, transform)) = camera.iter().next() {
+        commands
+            .entity(entity)
+            .insert(CameraController::with_speed_from_scale(
+                transform.translation.distance(Vec3::ZERO),
```

Comment:
Camera distance from the "center" feels like a pretty rough / arbitrary heuristic. Imagine a scene is a huge "town" and the camera is in the "players house", which lives "miles" away from from the origin. The camera would move uber fast in that scenario, despite the "scale" still being "human scale".

Maybe we should just make it a standard speed (for predictability), but add "speed up" and "slow down" shortcuts that scale the base walk/run speed by a configurable number?

Reply:
Yeah, I was chewing on that too. I think this is the way: there's not really any "objectively correct" way to determine the scale of a scene unfortunately.

Reply:
I removed the change on camera speed from this PR